scripts/create-new-actor.sh

```diff
@@ -67,25 +69,96 @@ namespace genny::actor {
 /**
  * TODO: document me
  *
+ * Indicate what the Actor does and give an example yaml configuration.
+ * Markdown is supported in all docstrings so you could list an example here:
+ *
+ * ${q}${q}${q}yaml
+ * SchemaVersion: 2017-07-01
+ * Actors:
+ * - Name: ${actor_name}
+ *   Type: ${actor_name}
+ *   Phases:
```

Comment:
Yup I agree the right documentation isn't in the right places at the moment and fixing this would definitely help the onboarding. I've added this to the genny ideas inbox.